### ISSUE TITLE
mimikatz_command -f's help menu speaks French

### ISSUE THREAD

**Issue by wchen-r7** [rapid7/meterpreter#119]
I think we forgot to translate this:
https://github.com/rapid7/meterpreter/blob/6d43284689240f4261cae44a47f0fb557c1dde27/source/extensions/mimikatz/modules/mod_mimikatz_standard.cpp#L8

When you supply something invalid for mimikatz_command -f, it will speak French.

**Comment by Meatballs1** [comment]
Ideally should deprecate the mimikatz extension, but it can still be useful as kiwi doesn't expose all the features yet. *whip* OJ.

**Comment by OJ** [comment]
Yeah yeah I know! :P Imma doing ma best! I'll get to it, I promise.

BTW, mimikatz will remain in action so long as WinXP/Win2k2 remain present. Mimikatz 2.0 (and therefore the Kiwi extension) rely on updated APIs which aren't present in older systems.
